webpush_channels/tests/test_views_channel.py

```diff
@@ -124,3 +124,15 @@ def test_push_notification_can_take_a_payload(self):
             webpusher_mock.assert_called_with(self.subscription)
             webpusher_mock.return_value.send.assert_called_with(
                 data=MINIMALIST_PAYLOAD['data'], ttl=15)
+
+    def test_invalid_encryption_keys_shows_error(self):
+        CHANGED_SUBSCRIPTION = MINIMALIST_SUBSCRIPTION
```

Comment:
You need to copy the value of the MINIMALIST_SUBSCRIPTION into CHANGED_SUBSCRIPTION because here you are also changing MINIMALIST_SUBSCRIPTION (which share the same memory address as MINIMALIST_SUBSCRIPTION)